library/Zend/View/Helper/PartialLoop.php

```diff
@@ -74,4 +94,51 @@ public function getPartialCounter()
     {
         return $this->partialCounter;
     }
+
+    /**
+     * Set object key in this loop and any child loop
+     *
+     * @param string $key
+     */
+    public function setObjectKey($key)
```

Comment:
That method overrides `setObjectKey()` in Partial, which we inherit. Maybe I should've added that to the docblock, but I'm unsure of the syntax.